docs/content/en/docs/concepts/tracing-policy/selectors.md

```diff
@@ -897,6 +898,95 @@ broken.
 
 Socket tracking is only available on kernel >=5.3.
 
+### Notify Killer action
+
+The `NotifyKiller` action notifies the killer program to kill or override syscall.
+
+The specs needs to have `killer` program definition, that instructs tetragon to load
+`killer` program and attach it to specified syscalls.
+
+```yaml
+spec:
+  killers:
+  - syscalls:
+    - "list:dups"
+```
+
+The syscalls expects list of syscalls or `list:XXX` pointer to list.
+
+Note that currently only single killer definition is allowed.
+
+
+The `NotifyKiller` action takes 2 arguments.
+
+```yaml
+matchActions:
+- action: "NotifyKiller"
+  argError: -1
+  argSig: 9
+```
+
+If pecified the argError will be passed to `bpf_override_return` helper to override the syscall return value.
+If pecified the argSig will be passed to `bpf_send_signal` helper to override the syscall return value.
+
+The following is spec for killing `/usr/bin/bash` program whenever it calls `sys_dup` or `sys_dup2` syscalls.
+
+```yaml
+spec:
+  lists:
+  - name: "dups"
+    type: "syscalls"
+    values:
+    - "sys_dup"
+    - "sys_dup2"
+  killers:
+  - syscalls:
+    - "list:dups"
+  tracepoints:
+  - subsystem: "raw_syscalls"
+    event: "sys_enter"
+    args:
+    - index: 4
+      type: "uint64"
+    selectors:
+    - matchArgs:
+      - index: 0
+        operator: "InMap"
+        values:
+        - "list:dups"
+      matchBinaries:
+      - operator: "In"
+        values:
+        - "/usr/bin/bash"
+      matchActions:
+      - action: "NotifyKiller"
+        argSig: 9
+```
+
+Note the `NotifyKiller` with killer program is meant to be used only on kernel versions
+with no support for fast attach of multiple kprobes (`kprobe_multi` link).
```

Comment:
Would it be possible to move that to the very beginning, after the "The `NotifyKiller` action notifies the killer program to kill or override syscall." sentence? Also, could we have a little more details on the why of this feature? Why this is interesting and maybe useful for the reader?

Reply:
ok, I'll add some description at the beginning, thanks